app/views/posts/show.html.erb

```diff
@@ -1 +1,30 @@
-<h1>Here is a list of posts for a given user</h1>
\ No newline at end of file
+<div class="show_post_view">
+    <div class="post-container">
+        <div class="post-details">
+        <div class="post-title"><%= @post.title %></div>
+        <div class="post-author">Author: <%= @post.author.name %></div>
+        <div class="post-comments">Comments: <%= @post.comments_counter %></div>
+        <div class="post-likes">Likes: <%= @post.likes_counter %></div>
+        </div>
+        <div class="post-content">
+        <div class="post-text"><%= @post.text %></div>
+        <div class="post-date">Date: <%= @post.created_at %></div>
+        </div>
+    </div>
+    <div class="post-comments">
+        <div class="comments-container">
+        <% @post.comments.each do |comment| %>
```

Comment:
* As shown in the wireframe below you are to render 5 most recent comments as you have created the method already in your post model, Kindly ensure your views function as shown in the wireframe below 👇 
![image](https://user-images.githubusercontent.com/52497246/222759568-830cefc5-4e6d-4520-a720-f848a71c8847.png)